RobotEV3/src/main/java/de/fhg/iais/roberta/visitor/codegen/utilities/Ev3VisitorHelper.java

```diff
@@ -0,0 +1,92 @@
+package de.fhg.iais.roberta.visitor.codegen.utilities;
+
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import de.fhg.iais.roberta.components.Configuration;
+import de.fhg.iais.roberta.components.ConfigurationComponent;
+
+public class Ev3VisitorHelper {
+	
+	public static String byteToHex(byte b) {
+	      // Returns hex String representation of byte b
+	      char hexDigit[] = {
+	         '0', '1', '2', '3', '4', '5', '6', '7',
+	         '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'
+	      };
+	      char[] value = { hexDigit[(b >> 4) & 0x0f], hexDigit[b & 0x0f] };
+	      return new String(value);
+	   }
+	
+	public static String getBytes(byte[] value) {
+		String byteHex = new String();
+	    for (int i = 0; i < value.length; i++) {
+	    	byteHex += "\\x" + byteToHex(value[i]);
+	    }
+	    return byteHex;
```

Comment:
Maybe drop the helper and use a StringBuilder like in
https://stackoverflow.com/questions/2817752/java-code-to-convert-byte-to-hexadecimal/50846880